massa-protocol-worker-2/src/handlers/block_handler/mod.rs

```diff
@@ -83,7 +83,7 @@ impl BlockHandler {
         });
 
         let block_propagation_thread = std::thread::spawn({
-            let _active_connections = active_connections.clone();
+            let _active_connections = active_connections;
```

Comment:
Is this line really needed ? Couldn't it be deleted ?

Reply:
Removed